crates/sui-move-build/src/linters/self_transfer.rs

```diff
@@ -135,21 +135,23 @@ impl SimpleAbsInt for SelfTransferVerifierAI {
             .any(|(addr, module, fun)| f.is(addr, module, fun))
         {
             if let Value::SenderAddress(sender_addr_loc) = args[1] {
-                let msg = format!(
-                    "Transfer of an object to transaction sender address in function {}",
-                    self.fn_name
-                );
-                let uid_msg =
-                    "Returning an object from a function, allows a caller to use the object \
+                if is_wrappable_obj_type(&f.arguments[0].ty) {
+                    let msg = format!(
+                        "Transfer of an object to transaction sender address in function {}",
```

Comment:
I know this was already here, but feels a bit strange to name the function we are in here, since it is always a local, per function analysis? Or is that not the case?

Reply:
Point taken - shortened up the message.